### PR TITLE
This function returns Json format of intent-template sentence data

### PR DESCRIPTION
**Pull request by agoz17**
This function creates JSON format of intent- template message from our database. This function will be used to send training data to wit.api

### CODE REVIEW

File: bookbotproject/witapi.py [hunk 1]
```diff
@@ -30,5 +30,11 @@ def auto_train_by_File(file_):
 	for sentence in sentences:
 		client.message(sentence)
```
Comment (theTytoAlba) [inline bookbotproject/witapi.py]: Can you please add a comment line here and explain what the template_feed_get_json  function returns briefly? Just to make things more clear for future use. :)

**Review by theTytoAlba — COMMENTED** [review]
Great, thank you!

**Comment by agoz17** [comment]
I've created the necessary comments as you reminded @theTytoAlba.

**Comment by theTytoAlba** [comment]
Very nice explanation of the function, thank you! I am merging this branch.

**Pull request merged**